sites/fast-creator/app/configs/fluent-ui/library.fluent-ui.registry.ts

```diff
@@ -0,0 +1,13 @@
+import { DesignSystem } from "@microsoft/fast-foundation";
+import { allComponents } from "@fluentui/web-components";
+import { setupFluentUIComponentDesignSystem } from "./library.fluent-ui.design-system.mapping";
+
+export function registerFluentUIComponents(): void {
+    setupFluentUIComponentDesignSystem(document.body);
+
+    DesignSystem.getOrCreate().register(
```

Comment:
See comment above on setting prefix. Also, if you have to set this to "fluent" that seems like a bug in our fluent web components. I think I know what's going on there. I'll put together a PR to fix it in the source.

Reply:
Changed to withPrefix for now. Can you link the PR as a comment to this when ready? I'll follow and remove the withPrefix once the fix is in.

Reply:
We merged the PR for FAST. I'll be submitting the PR for Fluent today. I'll link when I've got it up. Essentially, you'll be able to do something like this:

```ts
provideFluentDesignSystem().register(allComponents);
```

Here's the link to the FAST PR that was merged, for reference https://github.com/microsoft/fast/pull/4973

Reply:
Here's the related Fluent PR: https://github.com/microsoft/fluentui/pull/19056

Reply:
When that PR is merged and the package is published we will update the registration logic. Thanks @EisenbergEffect!